src/server/capacity_unit_calculator.h

```diff
@@ -55,6 +64,9 @@ class capacity_unit_calculator : public dsn::replication::replica_base
 #endif
 
 private:
+    void count_read_data(const dsn::blob &key, key_type type, int64_t size);
+    void count_write_data(const dsn::blob &key, key_type type, int64_t size);
```

Comment:
Is function `count_wirte_data` used for add cu and capture key for hotkey detection?